samples/msal-node-samples/custom-INetworkModule-and-network-tracing/HttpClientCurrent.ts

```diff
@@ -53,7 +53,17 @@ class NetworkUtils {
 /**
  * This class implements the API for network requests.
  */
-export class HttpClient implements INetworkModule {
+export class HttpClientCurrent implements INetworkModule {
```

Comment:
naming? Maybe `AxiosHttpClient` ?

Reply:
Maybe `HttpClientNative` would be more apt?